lib/src/upgrader.dart

```diff
@@ -82,6 +82,14 @@ class Upgrader {
   /// The upgrade dialog style. Used only on UpgradeAlert. (default: material)
   UpgradeDialogStyle dialogStyle;
 
+  /// Custom Dialog Builder for dialogStyle == DialogStyle.custom
+  Widget Function(
+    BuildContext context,
+    String title,
+    String message,
+    String? releaseNotes,
+  )? customDialogBuilder;
```

Comment:
The three buttons are missing: Ignore, Later, and Update Now.

Reply:
Added ✅